TileFormats/VectorData/README.md

```diff
@@ -1,19 +1,198 @@
 # Vector Data
 
-* [Draft spec](https://github.com/AnalyticalGraphicsInc/3d-tiles/pull/124/files)
-* GitHub issue, [#25](https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/25)
+## Contributors
 
-## Notes
+* Dan Bagnell, [@bagnell](https://github.com/bagnell)
+* Rob Taglang, [@lasalvavida](https://github.com/lasalvavida)
+* Patrick Cozzi, [@pjcozzi](https://twitter.com/pjcozzi)
 
-**Use cases**: Traditional geospatial features: points, polylines, and polygons.  Replacing KML.
+## Overview
 
-**Format**
-* Combination of binary (for positions, normals, etc.) and JSON (for labels, other metadata, etc.).
-* Concise representations for Cesium's [set of geometries](http://cesiumjs.org/2013/11/04/Geometry-and-Appearances/), including extrusions, and billboards and labels.
-   * Need to carefully select the representation for the best trade-off between conciseness and runtime processing.  For example, polygons will likely be pre-triangulated since it only adds indices to the payload, but will be subdivided at runtime since subdivision is fast and increases the vertex payload significantly.
-   * RTC positions for high precision-rendering.
-   * Context-aware compression.
-   * Bounding volume may need to be adjusted at runtime for terrain clamping.
-* Metadata for cracking, morphing, and perhaps label declutter.
+The _Vector_ tile format allows streaming of vector datasets including points, polylines, and polygons.  Points can be represented with a combination of billboard, label, and point graphics primitives.
 
-Could also name this a `Geometry` tile.
+Each point, poyline, and polygon is a _feature_ in the 3D Tiles specification language. 
+
+## Layout
+
+A tile is composed of two sections: a header immediately followed by a body.
+
+**Figure1**: Vector tile layout.
+![layout](figures/layout.jpg)
```

Comment:
The positions are stored separately because the polygon vertices need to be contiguous in the buffer in counter clockwise order. The indices are for the triangulated polygon.

Another reason is polylines and points won't need indices this way.

>Is this because they are all mapped to different Cesium primitives at runtime?

This used to be the case. The buffers were sent directly to the GPU and rendered, but that isn't the case anymore.